File: ALGORITHMS/Python/compare_the_triplets.py
def compare_the_triplets(a,b):
    length_data=len(a)
    sumA=0
    sumB=0
    for i in range(length_data):
        if a[i]>b[i]:
            sumA+=1
        elif a[i]<b[i]:
            sumB+=1
    return [sumA,sumB]

File: ALGORITHMS/Javascript/compare_the_triplets.js
const compare_the_triplets=(a,b)=>{
    const length_data=a.length;
    let sumA=0
    let sumB=0

    for (let index = 0; index < length_data; index++) {
        if (a[index]>b[index]){
            sumA+=1
        }else if(a[index]<b[index]){
            sumB+=1
        }
    }
    return [sumA,sumB]
}